src/components/viewDatabase/Sidebar.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import ToolbarStyle from './Sidebar.module.css';
+import JoinList from './JoinList';
+import DatabaseUIProvider from '../../providers/DatabaseUIProvider';
+import { Store } from "orbit-db-store";
+
+const SideBar: React.FC<{
+  joinEvents: string[],
+  selectJoin(join: string): void,
+  type: string,
+  store: Store,
+  goHome(): void,
+  uiProvider: DatabaseUIProvider
+}> = ({ joinEvents, selectJoin, type, goHome, uiProvider, store }) => {
+  const [showAddFields, toggleAddFields] = useState(false);

```

Comment:
Might have to make this more generic as we gonna have other buttons soon

Reply:
I can handle it so you can simply resolve it if you don't feel like changing

Reply:
So you mean the goHome bit?

Reply:
Nah the showAddFields bit